src/extension.ts

```diff
@@ -214,7 +214,13 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
             } else {
                 properties.result = 'Failed';
                 errorData = new ErrorData(err);
-                vscode.window.showErrorMessage(errorData.message);
+                if (errorData.message.includes('\n')) {
+                    util.getOutputChannel().appendLine(errorData.message)
+                    vscode.window.showErrorMessage('An error has occured. Check output log for more details.');
```

Comment:
nit: Can this be output _window_ instead of _log_? If it just says 'log', then users might try to find a log file

Reply:
Your wish is my command 👑